lib/mongoose_push/service/fcm/pool/supervisor.ex

```diff
@@ -24,6 +24,12 @@ defmodule MongoosePush.Service.FCM.Pool.Supervisor do
       )
     ]
 
+    :telemetry.execute(
+      [:mongoose_push, :fcm_supervisor, :init],
```

Comment:
`init` events should always get their `terminate` counterpart in order to enable metrics like "number of active instances".

Reply:
Here we use `Supervisor` and not `GenServer` and it doesn't have a `terminate` callback - not sure how to handle that